ang/civiawards.ang.php

```diff
@@ -12,7 +12,7 @@
 use CRM_Civicase_Helper_OptionValues as OptionValuesHelper;
 
 $options = [
-  'awardTypes' => 'civiawards_award_type',
+  'awardSubtypes' => 'civiawards_award_type',
```

Comment:
You forgot to rename the option group here: should it not be 
```
'awardSubtypes' => 'civiawards_award_subtype', ?
```

Reply:
This is an older commit, the current value is `civiawards_award_subtype`.